README.md

```diff
@@ -31,6 +24,13 @@ See the API documentation [for
 Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5/scala/index.html#package) and [for
 PySpark](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/index.html).
 
+| Topics  | Links                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+|:--------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Build   | [![Build Status](https://msdata.visualstudio.com/A365/_apis/build/status/microsoft.SynapseML?branchName=master)](https://msdata.visualstudio.com/A365/_build/latest?definitionId=17563&branchName=master) [![codecov](https://codecov.io/gh/Microsoft/SynapseML/branch/master/graph/badge.svg)](https://codecov.io/gh/Microsoft/SynapseML)                                                                                                                                    |
```

Comment:
Can we change New to Contact, put it last in the table, and add a button that says "contact | support" that has a "mailto:synapseml-support@microsoft.com" as the href?